docs/README_JP.md

```
@@ -1,10 +1,11 @@
-<p align="center">
+<div style="text-align: center;">
   <img
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)